kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/MetadataRequestsRetryRef.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.server.metadata
+
+import java.util.concurrent.ConcurrentLinkedQueue
+import java.util.concurrent.atomic.AtomicInteger
+
+class MetadataRequestsRetryRef {
+  private[metadata] val retryCount = new AtomicInteger(0)
+
+  private[metadata] val retryingTaskCount = new AtomicInteger(0)
```

Comment:
what is the diff？

Reply:
it's odd, if we want an overall control for retrying, it might be good to move it to MetadataManager

Reply:
done